sdk/formrecognizer/test-resources.json

```diff
@@ -60,6 +66,7 @@
         }
     },
     "variables": {
+        "roleDefinitionId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/a97b65f3-24c7-4388-baec-2e87135dc908')]",
```

Comment:
ID (a97b65f3-24c7-4388-baec-2e87135dc908) obtained from: https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles